experimental/classify.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+TASK_NAME=cola
```

Comment:
squad or cola here?

Reply:
I'm not using these files, so I think we should skip experimental files for now.